convene-web/app/lib/utilities.rb

```diff
@@ -0,0 +1,25 @@
+# Utilities are External Services that provide functionality to a piece of
```

Comment:
This comment immediately makes me think "Should we call this an `ExternalService` then?". Naming is hard, and I am myself a fan of using "Utility" when I can think of anything better, but I also recognize it is kinda like calling something "Thingie" -- not a huge value add. 😹